main/webapp/modules/core/scripts/project/history-panel.js

```diff
@@ -288,9 +288,34 @@ HistoryPanel.prototype._showApplyOperationsDialog = function() {
   
   elmts.dialogHeader.html($.i18n('core-project/apply-operation'));
   elmts.or_proj_pasteJson.html($.i18n('core-project/paste-json'));
+
+  elmts.operationJsonButton.on('click', async function() {
+    const input = document.createElement('input');
+    input.type = 'file';
+    input.accept = '.json'; // optional file types
+    input.onchange = async function() {
+      const file = input.files[0];
+      const reader = new FileReader();
+      reader.onload = function(e) {
+        const fileContent = JSON.parse(e.target.result);
+        const textAreaElement = document.querySelector('#textareaId')
+        if (textAreaElement) {
+          textAreaElement.textContent = JSON.stringify(fileContent, null, 2)
+
+        }
+      };
+      reader.addEventListener('error', function() {
+        alert('Error : Failed to read file');
```

Comment:
This would need to be localized with `$.i18n`, and in English there should not be any space between `Error` and the colon `:`.

Reply:
Done.

Reply:
No, it does not look like this string is localized. The string is still hard-coded in English. Can you localize it with `$.i18n`?